src/scss/components/_sidenav.scss

```diff
@@ -27,8 +27,6 @@
 
 .usa-sidenav__sublist a:not(.usa-button) {
   padding-left: units(4);
-  padding-top: units(1.5);
-  padding-bottom: units(1.5);
```

Comment:
Again not relevant for primary navigation, but in trying to find a canonical reference for the menu, I noticed a small difference in our sublist styling from the Figma reference. Previously, sublist items had smaller padding, but in the [Figma design](https://www.figma.com/file/kXLUewllU5YoQ6xG2fhwOM/Login-Design-System-LGDS-Figma-Library?node-id=4%3A2089) they're identical padding.

<table><thead><tr><th>Before</th><th>After</th><tr></thead><tbody><tr><td valign="top"><img src="https://user-images.githubusercontent.com/1779930/122229479-c8848680-ce86-11eb-8af7-774e0e0cd7c9.png" alt="before"></td><td valign="top"><img src="https://user-images.githubusercontent.com/1779930/122229530-d4704880-ce86-11eb-80cd-cfb91abbd629.png" alt="after"></td></tr></tbody></table>